src/Lifi.ts

```diff
@@ -254,6 +254,23 @@ export default class LIFI {
     return route
   }
 
+  /**
+   * Stops the execution of an active quote.
+   * @param {Step} step - A route that is currently in execution.
+   * @return {Route} The stopped route.
+   */
+  stopQuoteExecution = (step: Step): Route => {
+    const currRoute = convertStepToRoute(step)
+
+    if (!currRoute) {
+      throw new ValidationError(
+        "Missing 'fromAmountUSD' | 'toAmountUSD' in step estimate."
+      )
+    }
```

Comment:
We throw from `convertStepToRoute` if the validation does not pass, so this piece is unreachable and should be unnecessary. 👀

Reply:
Linked to this comment: https://github.com/lifinance/sdk/pull/115#discussion_r1109572456